<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QDialog" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
		background-color: black;
		border: 1px solid black;
		padding: 5px; 	
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lmovieinfo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Movie Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="ltitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>movie</string>
   </property>
  </widget>
  <widget class="QLabel" name="lduration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>duration</string>
   </property>
  </widget>
  <widget class="QLabel" name="lgenere">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>genere</string>
   </property>
  </widget>
  <widget class="QLabel" name="lyear">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>year</string>
   </property>
  </widget>
  <widget class="QLabel" name="lcountry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>country</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldirector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Director</string>
   </property>
  </widget>
  <widget class="QLabel" name="lstars">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Stars</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldirectorname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>directro name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lactor1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>actor1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lactor2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>actor2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lactor3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>actor3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lcontent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: yellow;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>content</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
